opentech/static_src/src/app/src/containers/GroupByStatusDetailView.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux'
+
+import DisplayPanel from '@components/DisplayPanel';
+import DetailView from '@components/DetailView';
+import ByStatusListing from '@containers/ByStatusListing';
+import { setCurrentSubmissionRound } from '@actions/submissions';
+
+class GroupByStatusDetailView extends React.Component {
+    componentWillMount() {
+        this.props.setSubmissionRound(this.props.roundId);
+    }
+
+    render() {
+        const passProps = {
+            listing: <ByStatusListing />,
+            display: <DisplayPanel />,
```

Comment:
DisplayPanel wont vary by View - it is always looking for the active application. DetailView should just hard code that and allow for the listing to be configured (how we select the active application)